README.md

```diff
@@ -1,11 +1,32 @@
-# lambda-template
-A GitHub template for quickly starting a new AWS lambda project.
+# lambda-mips-api
+An AWS Lambda microservice presenting MIPS chart of accounts data
 
-## Naming
-Naming conventions:
-* for a vanilla Lambda: `lambda-<context>`
-* for a Cloudformation Transform macro: `cfn-macro-<context>`
-* for a Cloudformation Custom Resource: `cfn-cr-<context>`
+## Architecture
+
+![Component Diagram](docs/lambda-mips-api_components.drawio.png)
+
+## Required Secure Parameters
+
+User credentials for logging in to the finance system are stored as secure parameters with a configurable prefix.
+By default, the prefix is `/lambda/mipsSecret`, resulting the following requried secure parameters:
+* `/lambda/mipsSecret/user`
+* `/lambda/mipsSecret/pass`
+
+## Template Parameters
+
+The following template parameters are set as environment variables in the lambda environment:
+| Template Parameter | Environment Variable | Description |
+| --- | --- | --- |
+| SsmParamPrefix | SsmPath | Path prefix for secure parameters |
+| MipsOrganization | MipsOrg | Log in to this organization in the finance system |
+
+## Respones Format
+
+The API will return a json string representing a dictionary mapping program codes to their names.
```

Comment:
can we add an example of how to call this API?  something like this.. https://github.com/Sage-Bionetworks-IT/lambda-batch-trigger#triggering

Reply:
done